src/site/content/en/fast/codelab-extract-and-inline-critical-css/index.md

```diff
@@ -0,0 +1,198 @@
+glitch: critical-css-starter
+date: 2019-05-08
+related_post: tools-for-extracting-critical-css
+---
+
+Whether you use a UI library or handcraft your styles, shipping a significant amount of CSS delays rendering because the browser must download and parse CSS files before it can show the page.
+
+This responsive ice cream gallery is built with [Bootstrap](https://getbootstrap.com/). UI libraries like Bootstrap speed up the development, but that often comes at the expense of bloated and unnecessary CSS which can slow down your load times. Bootstrap 4 is 187 KB, while [Semantic UI](https://semantic-ui.com/), another UI library, is a whopping 730 KB uncompressed. Even when minified and gzipped, Bootstrap still weighs around 20 KB, well over the [14 KB threshold](/tools-for-extracting-critical-css/#14KB) for the first roundtrip. 
+
+[Critical](https://github.com/addyosmani/critical) is a tool that extracts, minifies and inlines [above-the-fold](/tools-for-extracting-critical-css) CSS. This allows above-the-fold content to be rendered as soon as possible, even if CSS for other parts of the page has not yet loaded. In this codelab, you’ll learn how to use Critical's npm module. 
+
+## Measure
+
+Click **App** and then click **Show** to open the project without the code editor.
+
+To run a Lighthouse audit on this page:
+
+1. Press **Control+Shift+J** or **Command+Option+J** (Mac) to open DevTools.
+
+2. Click the **Audits** tab. 
+
+3. Make sure the **Mobile** radio button is selected. 
+
+4. Make sure the **Performance** checkbox is enabled. You can disable the rest of the checkboxes in the Audits section.
+
+5. Make sure the **Simulated Fast 3G, 4x CPU Slowdown** radio button is selected. 
+
+6. Make sure that the **Clear Storage** checkbox is enabled. With this option selected, Lighthouse will not load resources from the cache, which simulates how first-time visitors would experience the page.
+
+7. Click **Run Audits**. 
+
+![Audits panel of Chrome DevTools, powered by Lighthouse](lighthouse-audits.png)
+
+When you run an audit on your machine, the exact results may vary, but in the filmstrip view, you’ll notice the app has a blank screen for quite a while before finally rendering the content. This is why [First Contentful Paint](https://web.dev/first-contentful-paint/) (FCP) is high and why overall performance score is not great.
+
+<img src="lighthouse-audit-before.png" alt='Lighthouse audit showing performance score of 84, FCP 3 seconds and a filmstrip view of loading the app' class="w-screenshot">
+
+Lighthouse is here to help you fix performance issues, so look for solutions in the "Opportunities" section. “Eliminate render-blocking resources” is listed as an opportunity and that’s where Critical shines!
+
+
+<img src="eliminate-render-blocking-resources.png" alt='Lighthouse audit "Opportunities" section listing “Eliminate render-blocking resources”' class="w-screenshot">
```

Comment:
Was wondering if you have a higher res version of this? The current version might be fine, but if I recall you were generating a higher res copy for the related guide so maybe you could add a copy here as well?